apps/bundle_deploy/README.md

```diff
@@ -65,3 +65,20 @@ This will:
 - Build a `bundle_static.o` object containing the runtime functions
 - Build a `demo_static` executable which has static link to `bundle_static.o` and
   `model.o`, functions on a cat image, then prints the output results.
+
+
+### Cross compilation
+
+Example for dynamic linking for RISC-V qemu.
+
+```bash
+CC=riscv64-unknown-linux-gnu-gcc \
+CXX=riscv64-unknown-linux-gnu-g++ \
+TVM_TARGET="llvm -device=riscv_cpu -mtriple=riscv64-unknown-linux-gnu" \
```

Comment:
Does ```-device=riscv_cpu``` exists in TVM ?

Reply:
Yes, it was added in the PR https://github.com/apache/tvm/pull/8860